mne_bids/write.py

```diff
@@ -821,6 +881,23 @@ def write_raw_bids(raw, bids_basename, output_path, events_data=None,
         inferred from the stim channel using `mne.find_events`.
     event_id : dict | None
         The event id dict used to create a 'trial_type' column in events.tsv
+    anonymize : dict | None
+        If a dictionary is passed, data will be anonymized; identifying data
+        structures such as study date and time will be changed.
+        `daysback` is a required argument and `keep_his` is optional, these
+        arguments are passed to :func:`mne.io.anonymize_info`.
+
+        `daysback` : int
+            Number of days to move back the date. To keep relative dates
+            for a subject use the same `daysback`. According to BIDS
+            specificiations, the number of days back must be great enough
+            that the date is before 1925 thus `daysback` will be subtracted
+            from Dec 31, 1924.
```

Comment:
```suggestion
```

didn't we also agree to give the users a "rule of thumb" so that they don't hit an error on their first try. Something like, if daysback is between "blah" and "blah" you will anonymize properly

Reply:
It's any positive number, turns out to be less than around 8900. I just raise an error if they go over that. I figure most people will pick small numbers anyway.

Reply:
ahhh I didn't realize that `daysback` is now relative to Dec 31, 1924 instead of `meas_date`. You might want to undo my suggestions above then. It might be confusing for many people who are used to the MNE API. I think it should be spelt out really clearly in the documentation

Reply:
I think actually it better be in relation to meas_date, or I think meas_date - 100 years

Reply:
maybe add `yearsback` which defaults to 100? So it's clear you shift back 100 years and x days where is x is `daysback`. Wdyt?

Reply:
Ok now I changed it so that it's really clear that it's daysback with 100 years added. If the dataset that is to be anonymized spans more than 24 years, it won't be able to be anonymized until the LONGLONG fix. Basically, in order to preserve the meas_dates they need to give a daysback value relative to that, if I did it like before relative to 1925, then I was deleting the meas_date info. It gives an error now with the range of daysback acceptable which anything around 0 should be fine, anything this decade will be something like -4000 to 4000.